src/emulator/initEmulators.ts

```diff
@@ -46,6 +46,18 @@ export const AdditionalInitFns: AdditionalInitFnsType = {
 
     return mapToObject(additionalConfigs);
   },
+  [Emulators.DATACONNECT]: async () => {
+    const dataDir = await promptOnce({
+      name: "dataDir",
+      type: "input",
+      default: "./postgresData",
```

Comment:
Should this be something within the `dataconnect` directory?

In the order of my preference:
- `./dataconnect/.dataconnect/pglite_data` (hidden folder, out of sight)
- `./dataconnect/pglite_data` (next to schema and connector sources)
- `./dataconnect-generated/pglite_data` (next to generated SDK's default path)

Reply:
That seems reasonable - went with the first option, except that we use the configured data connect source directory.

Reply:
Ended up refactoring this a bit because when you pass a default to inquirer, it forces blank answers to that default. Instead, we'll just ask if you want to enable persistence, and tell you what field to change if you want a different directory.